### PR TITLE
Azure quickstart tutorial

### LINKED ISSUE #399: Create Azure quickstart
### Description
Create Azure quickstart 

### User Story
As a user wanting to deploy Pantheon on Azure, I want a tutorial/How-to guide that defines the steps to deploy Pantheon on Azure, so that I can have a hosted instance of Pantheon running.

### Acceptance Criteria
- A How-To tutorial that takes a user through the end-to-end steps to get Pantheon on Azure, including:
- Info/link to free Azure instanceSpec/setup of VMManual setup of Pantheon on the VMBasic quickstart and validation

### PR DESCRIPTION
**Pull request by NicolasMassart**
## PR description
Added a tutorial page in documentation to deploy the quickstart on Azure Linux VMs.
## Fixed Issue(s)

fixes #399 
fixes a quick typo on Private-Network-Quickstart.md, missing "r" in "Enterprise"

### CODE REVIEW

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+To do so, simply navigate in the Azure portal to your resource group, the one we named **pantheon-quickstart**
+then click on the **Delete resource group** button.
+
+### I want to remove the virtual machine only.
+Then simply navigate in the Azure portal to your resource group, the one we named **pantheon-quickstart**
+then click on the virtual machine resource and click the **Delete** button.
+
+### I want to keep the VM but remove the nodes network.
+Navigate to the VM in your Azure portal and click the **connect** button that will give you the
+information to connect using SSH.
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: The issue is that it depends a lot, you can have an SSH connection in many ways, depending on OS (Linux/Mac/Windows), tool you have the habit to use(terminal, Putty, other), so I don't want to enter this maze. I think people entering here know how to do that at least. Will add this as a requirement then.

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+To do so, simply navigate in the Azure portal to your resource group, the one we named **pantheon-quickstart**
+then click on the **Delete resource group** button.
+
+### I want to remove the virtual machine only.
+Then simply navigate in the Azure portal to your resource group, the one we named **pantheon-quickstart**
+then click on the virtual machine resource and click the **Delete** button.
+
+### I want to keep the VM but remove the nodes network.
+Navigate to the VM in your Azure portal and click the **connect** button that will give you the
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: Ok

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+![Your deployment is complete screenshot](deployment-complete-screenshot.png)
+
+CLick on the resource named **quickstart** in this list and you'll land on the virtual
+machine overview page.
+
+## Accessing the Alethio EthStat-light explorer
+
+On the right part of the virtual machine page you can read the **Public IP address** of this machine. 
+Copy it and past it in your browser navigation bar.
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: common sens but yes, will add "new browser tab or window"

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+## Connect to Azure Portal
+Once you have created an account or if you already have one, login to [portal.azure.com](https://portal.azure.com/)
+
+## Creating resources
+Go on the home page and click on ![Create resource button](create-resource-button-screenshot.png)
+at the top left hand corner.
+
+Search the Marketplace for **Ubuntu Server 18.04 LTS by Canonical** and click on it then click the 
+blue **Create** button at the bottom.
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: ok

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+Fill the second field named **Command** with the following informations (replacing the pre filled `sh script.sh`):
+
+```bash
+sh install_and_run.sh azure
+```
+
+!!!important
+    The username is included at the end of this command, **azure** in our case, but change
+    to whatever you defined on the Basics step if you did not use the same as us.
+
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: ok

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+Navigate to the VM in your Azure portal and click the **connect** button that will give you the
+information to connect using SSH.
+Once connected to the machine, navigate to the pantheon-quickstart directory:
+```bash
+cd pantheon-quickstart
+``` 
+then run the remove script
+```bash
+./remove.sh
+```
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: There's a difference between stopping and removing the VM isn't there though?  I stopped but didn't remove the VM.  Then I needed the instructed to use the install_and_run.sh script to restart the quickstart after restarting the VM.

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+### Guest config
+This step aims at installing required software (Docker and Docker-compose) on your virtual machine 
+and retrieve and run the quickstart private network.
+
+To do so, click on the link named **Select an extension to install** and a new resource pane will
+appear on the right.
+
+In the list click on **Custom Script For Linux**, another pane opens with a blue **Create** button at
+its bottom, click on it.
```
Comment (donntrenton) [inline docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: Getting a failure on this step. A new pane opens, but it's blank with no controls at all. Also does this for other extensions. I've reproed this on Edge and Chrome, on different computers. My repro follows your script except that I selected username/password.

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (donntrenton) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: I like how you provided several options.

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+## Creating resources
+Go on the home page and click on ![Create resource button](create-resource-button-screenshot.png)
+at the top left hand corner.
+
+Search the Marketplace for **Ubuntu Server 18.04 LTS by Canonical** and click on it then click the 
+blue **Create** button at the bottom.
+
+A form will display, fill it with the following values for each step, starting with the **Basics** one:
+
+### Basics step
```
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: Well tabs are just graphical representation of a step.

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md (hunk 7 above)
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: and you have windows...

File: docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (NicolasMassart) [reply docs/Getting-Started/azure/Azure-Private-Network-Quickstart.md]: added how to start again after ./remove.sh

**Pull request merged**